CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Added
 
 - Added Italian to platform
+- Expanded PB: new currencies (tokens/credits), minimum budget, copy changes
```

Comment:
Would write it out a bit more elaborate (with sub items), as not everyone knows what `PB` means or gets the context of the change

```suggestion
- Improved participatory budgeting
  - Support for new virtual currencies (TOK: tokens, CRE: credits)
  - A minimum budget limit can be configured per project, forcing citizens to fill up their basket to some extent
  - Copy improvements
```

Reply:
I'd even go further and distinguish between min = max budget (which translates to a fixed amount of currency that must and can be spent at the same time), and min < max (which translates to a minimum that should be spent and that should stay below a certain budget

Reply:
(just added it)